### PR TITLE
update README for runtime/app agnostic language

### PR DESCRIPTION
**Pull request by ajm01**
new README removes mention of Liberty or Spring Boot as well as mentions of Docker support (for now)

working on the Tutorial next, but want to get this in for today

### CODE REVIEW

File: boost-maven/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### What is Boost?
+
+This is a prototype Maven plugin to package a fully configured Java EE or MicroProfile application with a target runtime.
+
+When added to your pom.xml, the plugin will
+
+1. Install the desired target runtime.
+2. Create a server.
+3. Install the application to the server.
+4. Configure the server appropriatly for the application.
```
Comment (cherylking) [inline boost-maven/README.md]: spelling - appropriately

**Comment by cherylking** [comment]
One more comment...you do not need the package phase...the package goal is already bound to that phase by default.

File: boost-maven/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    * `java -jar target/<application name>.jar`
+
+#### Quick start - uber jar
+
+1. Add the following to your project pom.xml
+    ```xml
+      <plugin>
+            <groupId>boost</groupId>
+            <artifactId>boost-maven-plugin</artifactId>
+            <version>0.1</version>
```
Comment (cherylking) [inline boost-maven/README.md]: Change to version 0.1.3-SNAPSHOT

Comment (ajm01) [reply boost-maven/README.md]: scott gave me some direction to remove the uber jar info, so took this out

File: boost-maven/README.md (hunk 1 above)
Comment (ajm01) [reply boost-maven/README.md]: fixed

File: boost-maven/README.md (hunk 2 above)
Comment (cherylking) [reply boost-maven/README.md]: I don't understand. The pom.xml still needs to include the boost-maven-plugin.

**Pull request merged**